tests/conftest.py

```diff
@@ -279,6 +278,8 @@ def corsika_output_file_name():
 
 @pytest.fixture
 def corsika_histograms_instance(io_handler, corsika_output_file_name):
+    from simtools.corsika.corsika_histograms import CorsikaHistograms
+
```

Comment:
shouldn't we add the pylint exception here or it doesnt matter for the test directory?

Reply:
This is not needed, as we do not run pylint over the test directory (we would get lots of complaints, e.g. from all the internal class functions we call in the test).

So no pylint comment required.